examples/android/BUILD

```diff
@@ -2,7 +2,7 @@ package(default_visibility = ["//visibility:public"])
 
 android_library(
     name = "firestore_config_manager_impl",
-    srcs = ["src/main/java/com/google/startupos/android/FirestoreInitializer.java"],
+    srcs = ["src/main/java/com/google/startupos/examples/android/FirestoreInitializer.java"],
```

Comment:
One doubt. When Bazel needs to reference these files since it cares about where the files are, how does it come to know where these files actually reside? Does it just start searching recursively since there is no other place where we are providing the actual path?

Reply:
I'm not sure about Bazel's internal implementation, but AFAIU, when you build a target, the references bazel resolves are to other Bazel targets, and the target itself contains the source files.
Once compiled, it's Java mechanism that finds classes (based on classpath), and there I believe it relies on package and class name.

Reply:
Cool. Sounds good. I have incorporated the changes. Kindly let me know if there is any other change that is required. Alongside, I'll even understand the basics of Bazel.

Reply:
Nope, all good